tests/static/all_question_types.xml

```diff
@@ -85,6 +85,7 @@
 					</text>
 					<text id="question1-help">
 						<value>question1 en help</value>
+						<value form="markdown">question1 en help</value>
```

Comment:
The spec says: 
> automatically treat all text in the ‘help’ text field as markdown by generating the xml for the xforms with duplicate <value form=”markdown”></value> fields

So I think that means it's intended for markdown & non-markdown text will be the same in all cases for now. For this release there isn't any pretty UI for making the markdown as it's only for TDH.

If the feature flag is set does that mean that we would only want to write the markdown form? I'm not sure how commcare would react if it was given <value form="markdown"> but not <value>